rfcs/2020-04-06-1858-automatically-adjust-limits.md

```diff
@@ -0,0 +1,164 @@
+# RFC 1858 - 2020-04-06 - Automatically Adjust Limits
```

Comment:
```suggestion
# RFC 1858 - 2020-04-06 - Automatically Adjust HTTP Request Limits
```

I think we should make it clear that this RFC focuses on HTTP based sinks only.

Reply:
This is true, and it's worth clarifying that this doesn't apply universally, but the HTTP part seems accidental. It is documented to apply at the level of the `tower` crate service builder, so any service that uses that framework would have this capability. Now, it happens that all of our HTTP services use the tower framework, and vice versa, but it isn't necessarily so.